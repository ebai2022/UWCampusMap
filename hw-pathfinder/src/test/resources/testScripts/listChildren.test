# Tests the different behavior of list children

CreateGraph g

AddNode g n1
AddNode g n2
AddNode g n3

# No children
ListChildren g n1

# One child
AddEdge g n1 n2 5.0
ListChildren g n1

# Multiple edges from same node
AddEdge g n1 n2 10.0
ListChildren g n1

# Multiple edges from same and different nodes
AddEdge g n1 n3 3.0
ListChildren g n1