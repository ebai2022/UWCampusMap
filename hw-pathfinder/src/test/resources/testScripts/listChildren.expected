# Tests the different behavior of list children

created graph g

added node n1 to g
added node n2 to g
added node n3 to g

# No children
the children of n1 in g are:

# One child
added edge 5.000 from n1 to n2 in g
the children of n1 in g are: n2(5.000)

# Multiple edges from same node
added edge 10.000 from n1 to n2 in g
the children of n1 in g are: n2(5.000) n2(10.000)

# Multiple edges from same and different nodes
added edge 3.000 from n1 to n3 in g
the children of n1 in g are: n2(5.000) n2(10.000) n3(3.000)